addons/sale_timesheet/tests/test_project.py

```diff
@@ -97,3 +97,52 @@ def test_fetch_sale_order_items(self):
         self.project_global.allow_billable = False
         self.assertFalse(self.project_global._get_sale_order_items())
         self.assertFalse(self.project_global._get_sale_orders())
+
+    def test_analytic_account_balance(self):
+        """
+            1) Add new billable project
+            2) Add Employee/SOL mapping in the project
+            3) Add Task and Timesheet with the same user
+            4) Assert analytic_account_balance is calculated
+        """
+        partner = self.env['res.partner'].search([('name', '=', 'Deco Addict')], limit=1)
+        employee = self.env['hr.employee'].search([], limit=1)
+        self.assertTrue(partner)
+        self.assertTrue(employee)
+
+        project = self.env['project.project'].create({
+            'name': 'Project A',
+            'allow_billable': True,
+            'partner_id': partner.id
+        })
```

Comment:
You should use the project create in a common class instead of a creating a new one.